src/util/Sort.js

```diff
@@ -335,6 +336,23 @@ export function getLabels(cube, sort) {
     return labels;
   } else if (sort == 'Unsorted') {
     return ['All'];
+  } else if (sort == 'Elo') {
+    var items = [];
+    cube.forEach(function(card, index) {
```

Comment:
Buckets?
- find elo range from cards (min to max)
- bucket_size = 10 * elo_range / num_cards_in_cube   
- bucket size min 1

Reply:
Just so it's written down somewhere, the Elo system should be centered at 1200 with a range of roughly 400-2000, so buckets can be made accordingly.

Reply:
I'll do buckets of 50, and just use math to divide it up